weasyl/define.py

```diff
@@ -329,18 +329,16 @@ def captcha_public():
 def captcha_verify(form):
     if config_read_bool("captcha_disable_verification", value=False):
         return True
-    if not form.recaptcha_response_field:
+    if not form.g_recaptcha_response:
         return False
 
     data = dict(
-        privatekey=config_obj.get(_captcha_section(), 'private_key'),
-        remoteip=get_address(),
-        challenge=form.recaptcha_challenge_field.encode('utf-8'),
-        response=form.recaptcha_response_field.encode('utf-8'))
-
-    response = http_post('https://www.google.com/recaptcha/api/verify', data=data)
-    result = response.text.splitlines()
-    return result[0] == 'true'
+        secret=config_obj.get(_captcha_section(), 'private_key'),
+        response=form.g_recaptcha_response['g-recaptcha-response'],
+        remoteip=get_address())
+    response = http_post('https://www.google.com/recaptcha/api/siteverify', data=data)
+    jsonResult = response.json()
+    return jsonResult['success']
```

Comment:
I'll clean up this variable name (Line 340/341) tomorrow to match the lower_with_underscore standard and push a commit when done. (Likely also rename it captcha_validation_result, or something slightly more descriptive.)